projects/igniteui-angular/src/lib/grids/summaries/summary-cell.component.ts

```diff
@@ -91,7 +96,46 @@ export class IgxSummaryCellComponent {
         return (this.column.grid as any);
     }
 
+    /**
+     * @hidden @internal
+     */
+    public isDateSummary(summaryKey: string): boolean {
+        return summaryKey === 'latest' || summaryKey === 'earliest';
+    }
+
     public translateSummary(summary: IgxSummaryResult): string {
         return this.grid.resourceStrings[`igx_grid_summary_${summary.key}`] || summary.label;
     }
+
+    /**
+     * @hidden @internal
+     */
+    public appySummaryPipe(summary: IgxSummaryResult): any {
+        const pipeArgs = this.column.pipeArgs;
+        const locale = this.grid.locale;
+        if (summary.key === 'count') {
```

Comment:
IMO we should transform with the pipe the `count` key only if it is from some of our summary operand types.